torch_xla_py/xla_model.py

```diff
@@ -237,7 +244,8 @@ def xla_loss(loss_fn, output_xla_tensors, labels):
             flat_tensors[flat_index].requires_grad = True
             replica_outputs.append(flat_tensors[flat_index])
             flat_index += 1
-        losses.append(loss_fn(*replica_outputs, labels[i]))
+        replica_outputs_and_label = _append_label_to_tensor_list(replica_outputs, labels[i])
```

Comment:
Line length check

Reply:
I'm running `autopep8` and it doesn't do anything to lines longer than 80 chars, I'm not sure what's going on.

Reply:
Maybe it's a g3 rule.
Would be nice to keep line lengths under control IMHO, even w/out the support of external tools.

Reply:
Ok, turns out it requires `--max-line-length 79` to be specified in order to enforce *any* line length, help is misguiding (says the default is 79). I'm going to send another PR which formats this file and tests as well.